### PR TITLE
Docs to show S3 and Local File Store support for SAM and Serverlesss

### PR DESCRIPTION
**Pull request by SushrutHarness**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ X] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ X] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

**Review by thisrohangupta — APPROVED** [review]

File: docs/continuous-delivery/deploy-srv-diff-platforms/aws/aws-sam-deployments.md [hunk 1]
```diff
@@ -123,14 +123,31 @@ To add your template, do the following:
...
+11. Select **Submit**.
+
+#### AWS S3
+8. Select the AWS S3 provider.
+9. Select or create a new Harness AWS connector to your AWS S3 provider, and then select **Continue**.
+10. In **Manifest Details**, enter the following:
+   1. **Manifest Identifier** Enter a name that identifies this Serverless manifest.
+   2. **Region** Select the AWS region to use for the connection.
+   3. **Bucket Name** Enter the AWS S3 bucket name.
+   4. **File/Folder Path** Enter the path from the root of the bucket to the serverless.yml file
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/aws/aws-sam-deployments.md:145]:
```suggestion
   4. **File/Folder Path** Enter the path from the root of the bucket to the `serverless.yml` file.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/aws/aws-sam-deployments.md [hunk 2]
```diff
@@ -123,14 +123,31 @@ To add your template, do the following:
...
+11. Select **Submit**
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/aws/aws-sam-deployments.md:146]:
```suggestion
11. Select **Submit**.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+title: Manifest Source for Serverless Lambda
+description: Learn configuration details for manifest sources for Serverless Lambda
+sidebar_position: 3
+---
+
+# Manifest Sources
+
+You will need to configure a manifest source in your service when deploying a Serverless Lambda function. This page will describe the possible sources and their configuration options.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md:9]:
```suggestion
You will need to configure a manifest source in your service when deploying a Serverless Lambda function. This topic describes the possible sources and their configuration options.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Before You Begin 
+
+These configuration options can be found when adding a manifest to a service with a **Deployment Type** of **Serverless Lambda**. 
+
+## Manifest Sources
+
+### Git Sources
+
+All git based sources have the same configuration paramters and hence will all be described here.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md:19]:
```suggestion
All Git-based sources have the same configuration parameters and hence will all be described here.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+Below describes setting configuration parameters for AWS S3 bucket as a manifest source. 
+
+#### Manifest Source
+
+1. **AWS Connector**: Your [AWS Connector](/docs/cloud-cost-management/use-ccm-cost-optimization/optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/add-connectors/connect-to-an-aws-connector/). Add or create one here.
+
+#### Manifest Details
+
+1. **Manifest Identifier** Enter a name that identifies this Serverless manifest.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md:42]:
```suggestion
1. **Manifest Identifier**: Enter a name that identifies this Serverless manifest.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+2. **Region** Select the AWS region to use for the connection.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md:43]:
```suggestion
2. **Region**: Select the AWS region to use for the connection.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+3. **Bucket Name** Enter the AWS S3 bucket name.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md:44]:
```suggestion
3. **Bucket Name**: Enter the AWS S3 bucket name.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+4. **File/Folder Path** Enter the path from the root of the bucket to the serverless.yml files.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md:45]:
```suggestion
4. **File/Folder Path**: Enter the path from the root of the bucket to the `serverless.yml` files.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/serverless-lambda-cd-quickstart.md [hunk 9]
```diff
@@ -285,6 +287,10 @@ Next, we can add a serverless.yaml for our deployment. We'll use [the publicly-a
 
 You can see the serverless.yaml manifest in Harness.
 
+:::info
+You can also use AWS S3 or Harness Local FileStore as your manifest provider. For information on how to configure those see [Manifest Providers for Serverless Lambda](/docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md).
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/serverless-lambda-cd-quickstart.md:291]:
```suggestion
You can also use AWS S3 or Harness Local File Store as your manifest provider. For information on how to configure those manifests, go to [Manifest Providers for Serverless Lambda](/docs/continuous-delivery/deploy-srv-diff-platforms/serverless/manifest-sources-for-serverless-lambda.md).
```

**Review by brian-f-harness — COMMENTED** [review]
Added a few minor suggested changes, mostly formatting and a couple of typos

**Pull request merged**